src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -14,6 +14,7 @@
                  i18n>Type</label>
           <div class="cd-col-form-input">
             <select id="service_type"
+                    name="service_type"
```

Comment:
Change reason: As `selectOption` in e2e  file requires `select[name=` to serach for a service name.